benefits/superuser.py

```diff
@@ -0,0 +1,23 @@
+import os
+import logging
+
+from django.contrib.auth import get_user_model
+
+logger = logging.getLogger(__name__)
+
+User = get_user_model()  # get the currently active user model
+
+username = os.environ.get("DJANGO_SUPERUSER_USERNAME")
```

Comment:
If the env vars are undefined, `os.environ.get(....)` hides this fact and the code continues along, only blowing up later.

I think we can use `os.environ[...]` here since these are all required values.